client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -117,7 +117,7 @@ export class ImageEditorToolbar extends Component {
 		const items = [
 			{
 				action: AspectRatios.FREE,
-				label: translate( 'Free' ),
+				label: translate( 'Free', {context: 'Crop option in image editor'} ),
```

Comment:
:disappointed: 43 existing translations will be lost with this change.
If the string must change, the following option has already been translated **40** times:
`translate( 'Free', { context: 'Jetpack Free Plan'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/emilyaudela-patch-2)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).